src/platform/silabs/SiWx917/SiWxPlatformInterface.h

```diff
@@ -21,7 +21,9 @@
 
 namespace {
 #if ENABLE_CHIP_SHELL && CHIP_CONFIG_ENABLE_ICD_SERVER
+#ifdef RTE_UULP_GPIO_1_PIN
```

Comment:
I understand that the other boards doesn't support the UULP 1 pin, but we have to ensure that feature i.e., sleepy with shell works with other devices as well. 
Please add a comment as well, or just change to the common unused pin and test it.